akka-docs/src/main/paradox/typed/logging.md

```diff
@@ -411,9 +411,20 @@ With Logback the ActorSystem name is available with `%X{sourceActorSystem}` spec
   </encoder>
 ```
 
-Akka's internal logging is asynchronous which means that the timestamp of a log entry is taken from
+The address of the actor system, containing host and port if the system is using cluster, is available through `akkaAddress`:
+
+```
+  <encoder>
+    <pattern>%date{ISO8601} %-5level %logger{36} %X{akkaAddress} - %msg%n</pattern>
+  </encoder>
+```
+
+
+For typed actors the log event timestamp is taken when the log call was made but for
+Akka's _internal_ logging as well as the classic actor logging is asynchronous which means that the timestamp of a log entry is taken from
 when the underlying logger implementation is called, which can be surprising at first.
-If you want to more accurately output the timestamp, use the MDC attribute `akkaTimestamp`.
+If you want to more accurately output the timestamp for such loggers, use the MDC attribute `akkaTimestamp`. Note that 
+the MDC key will not have any value for a typed actor.
```

Comment:
Good to mention this, though it might be nice to provide that value eventually (https://github.com/akka/akka/issues/28153), since we'll be mixing logging methods for quite a while

Reply:
Yeah we have to solve that at some point.